profiler/tools/stencil_reproducer.py

```diff
@@ -0,0 +1,135 @@
+import fnmatch
+import pickle
+from datetime import datetime
+from glob import glob
+from os import getcwd, getenv, listdir, mkdir, path, walk
+from shutil import copy, copytree
+from typing import Any, Dict, Tuple
+
+import numpy as np
+from gt4py import storage
+
+
+STENCIL_CANDIDATE_FOR_EXTRACT: Dict[str, Tuple[str, str]] = {}
```

Comment:
why are these global and not passed to the functions?

Reply:
Because this is used in the hook passed to `sys.setprofile` (field_serialization) and in the collector you can't pass kwargs to the hook function.

Though one is unused, so I removed it

Reply:
ah cool, thanks for the explanation